R/interface.R

```diff
@@ -261,11 +285,23 @@ dust_class <- function(alloc, run, reset, state, step, reorder,
         .Call(private$cpp_run, private$ptr, step_end)
       },
 
+      ## TODO: this probably needs changing as we can't set state and
```

Comment:
Can you add a test `is.null(private$data)` below? Or is NULL possible after data has been set?

Reply:
Sorry, that was a stale comment!